packages/core/src/progress/LinearProgress/LinearProgress.tsx

```diff
@@ -36,15 +36,17 @@ export interface LinearProgressProps extends ComponentPropsWithoutRef<"div"> {
   value?: number;
 }
 
+const INDETERMINATE_BAR_WIDTH = 66;
+
 export const LinearProgress = forwardRef<HTMLDivElement, LinearProgressProps>(
   function LinearProgress(
     {
       className,
       hideLabel = false,
       max = 100,
       min = 0,
-      value = 0,
-      bufferValue = 0,
+      value,
```

Comment:
Is changing the default values of these props relevant to document?

Reply:
Updated props table